MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -52,10 +52,19 @@ typedef struct {
   ///
 } SOFTWARE_STM_HEADER;
 
+// MU_CHANGE - [START]: Add new CPU information header for STM
 typedef struct {
-  MSEG_HEADER            HwStmHdr;
-  SOFTWARE_STM_HEADER    SwStmHdr;
+  UINT32    NumberOfCpus;
+  /// Pad to take up 1KB of space
+  UINT8     Reserved[SIZE_1KB - sizeof (UINT32)];
+} CPU_INFORMATION_HEADER;
+
+typedef struct {
+  MSEG_HEADER               HwStmHdr;
+  SOFTWARE_STM_HEADER       SwStmHdr;

```

Comment:
The order looks good now, but you're going to have to add reserved padding to the end of `SOFTWARE_STM_HEADER` so it is exactly 1KB in size and `CPU_INFORMATION_HEADER` begins at a 1KB boundary.